pkg/apis/storage/types.go

```diff
@@ -294,6 +294,23 @@ type CSIDriverSpec struct {
 	// +optional
 	PodInfoOnMount *bool
 
+	// Defines if the underlying volume supports changing ownership and
+	// permission of the volume before being mounted.
+	// If set to Always, SupportsFSGroup indicates that
+	// the volumes provisioned by this CSIDriver support volume ownership and
+	// permission changes, and the filesystem will be modified to match the
+	// defined fsGroup every time the volume is mounted.
+	// If set to Never, then the volume will be mounted without modifying
+	// the volume's ownership or permissions.
+	// Defaults to Evaluate, which results in the volume being examined
+	// and the volume ownership and permissions attempting to be updated
+	// only when the PodSecurityPolicy's fsGroup is explicitly defined, the
+	// fsType is defined, and the PersistentVolumes's accessModes is RWO.
+	// This field is alpha-level, and is only honored by servers
+	// that enable the CSIVolumeSupportFSGroup feature gate.
+	// +optional
+	SupportsFsGroup *SupportsFsGroup
```

Comment:
`SupportsFSGroup`, abbreviations are capitalized.

Reply:
I've updated all of these references to be `SupportsFSGroup`.